pull-request.py

```diff
@@ -102,9 +97,14 @@ def create_pull_request(
                 print("Pull request from %s to %s is already open!" % (source, target))
                 is_open = True
 
+                # Does the user want to pass if the pull request exists?
+                if os.environ.get("PASS_IF_EXISTS"):
+                    print("PASS_IF_EXISTS is set, exiting with success status.")
+                    sys.exit(0)
```

Comment:
```suggestion
                    sys.exit(0)
                break
```
What are your thoughts about putting a break from the loop as soon as we find the branch (and granted that we don't exit with 0, success)

Reply:
Makes sense!